packages/api/.env.default

```diff
@@ -21,33 +21,41 @@ JWT_RESET_SECRET=Production_is_secured_with_a_long_random_string
 JWT_FARM_SECRET=Here_we_can_use_friendly_explanations
 JWT_SCHEDULER_SECRET=Another_token_was_needed_for_the_scheduler
 
-OPEN_WEATHER_APP_ID=?
+# Create your own (free in most cases) Google API key at https://console.cloud.google.com/apis/dashboard
+# Same as the VITE_GOOGLE_MAPS_API_KEY under webapp
 GOOGLE_API_KEY=?
 
-GMAIL_API_CLIENT_ID=?
-GMAIL_API_CLIENT_SECRET=?
-GMAIL_ACCESS_TOKEN=?
-GMAIL_REFRESH_TOKEN=?
+# Create your own free or student OpenWeather API key at https://openweathermap.org/price
+# Same as the VITE_WEATHER_API_KEY under webapp
+OPEN_WEATHER_APP_ID=?
 
+# Create your own Google API key at https://developers.google.com/identity/oauth2/web/guides/get-google-api-clientid
 GMAIL_API_CLIENT_ID=?
 GMAIL_API_CLIENT_SECRET=?
+GMAIL_ACCESS_TOKEN=?
```

Comment:
Did we ever find out where GMAIL items came from? I still feel like we haven't found where these are at.

Reply:
No, I poked around and don't know where they came from. We also had them duplicated.

Reply:
Is there anyone we can ask about them?